content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,271 @@
+Teaching bubbles are a special [popover]() that calls attention to a feature in a specific part of the page. Generally, features highlighted by teaching bubbles should be used to educate the user and enriching to the user’s task at hand. Consider the page the user is currently viewing and what jobs-to-be-done they may be performing.
+
+|Pattern |Quickstart| Top user benefits|
+|:--|:--|:--|
+|Whent to use |<li>Call attention to new or important features</li><li>Invoked by the system, like a new capability being available</li>|The highlighted feature extends or improves the user's likely task at hand|
+|When not to use |<li>Don't point to hidden elements</li><li>If the user cannot immediately interact and benefit from the feature highlighted</li>|Do not distract the user from completing their task.|
+
+## General guidelines
+Refer to the [Primer popover component](https://primer.style/css/components/popover) for more detailed implementation guidelines.
+
+```html live title="Basic"
+<div class="Popover right-0 left-0 position-relative">
+  <div class="Popover-message text-left p-4 mt-2 mx-auto Box box-shadow-large">
+    <h4 class="mb-2">Teaching bubble heading</h4>
+    <p>Use this as a teaching moment that may enrich a user’s experience. Keep message to a maximum of 160 characters.</p>
+    <button class="btn btn-sm btn-outline mt-2" type="button">Primary action</button>
+    <button class="btn btn-invisible ml-2">Ok, dismiss</button>
+  </div>
+</div>
+```
+
+- Show one teaching bubble at a time.
+- Recommended to include a headline that states the purpose of the message.
+- Keep your message short and pithy, around 160 characters. 
+- If additional information must be added, consider a stepped guide or a “Learn more” link that opens in a new window or tab. 
+- It is preferable to use a descriptive dismiss button (e.g. “Ok, got it” or “Done”) over a close icon; if a close icon must be used, remember to apply the correct tab order and polite annoucements for screenreaders.
+-If a guided feature setup is needed in a recurring manner, consider using other more inline  patterns like empty states in lieu of teaching bubbles.
+- Keep your copy to maximum 160 characters.
+
+### Multiple calls to action
+The default dismissal button should say “Ok, got it” as a way to inject more personality while reaffirming with “ok” which internationalizes better as a more commonly recognized phrase. As a secondary call to action, it should be “Ok, dismiss.”
+
+<p><img src="https://user-images.githubusercontent.com/6744014/88741737-43b93e80-d0f5-11ea-8445-5b878607756e.png" width="180"/></p>
+
+If the primary action is not to dismiss the teaching bubble, they should invite the user to engage with the value proposition promised. Call to actions in teach bubbles should not take the user away from their current page. 
+
+Actions that take the user to a new page may only be linked from the copy and must open in a new tab or window to prevent loosing the user’s context.
```

Comment:
"If the call to action takes the user to a new page, it should be linked from the text and must open in a new tab or window to prevent losing the user's context."